docs/site/src/pages/Home.js

```diff
@@ -17,7 +17,7 @@ export const styleSheet = createStyleSheet('Home', (theme) => {
       flex: '1 0 100%',
     },
     hero: {
-      minHeight: '100%', // Makes the hero full height until we get some more content.
+      minHeight: '100vh', // Makes the hero full height until we get some more content.
```

Comment:
For iOS:
![capture d ecran 2016-12-04 a 16 56 10](https://cloud.githubusercontent.com/assets/3165635/20867338/9c5d24e8-ba42-11e6-8c7e-4064a1ea0ee6.png)